src/test/daml/Daml/Finance/Test/Util/Lifecycle.daml

```diff
@@ -0,0 +1,158 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
Do you need this?

Reply:
No, seems to work without it. I have removed it now.